### PR TITLE
{compiler,mpi,numlib,system,toolchain}[GCC/5.2.0] GCC v5.2.0, hwloc v1.11.3, OpenMPI v2.0.0, ...

### PR DESCRIPTION
**Pull request by shahzebsiddiqui**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/g/GCC/GCC-5.2.0.eb [hunk 1]
```diff
@@ -29,7 +28,7 @@ sources = [
 
 patches = [('mpfr-%s-allpatches-20150717.patch' % mpfr_version, '../mpfr-%s' % mpfr_version)]
 
-builddependencies = [('M4', '1.4.17')]
+# builddependencies = [('M4', '1.4.17')]
```
Comment (boegel) [inline easybuild/easyconfigs/g/GCC/GCC-5.2.0.eb]: @shahzebsiddiqui this change should not be here; same for the other changes in this easyconfig

File: easybuild/easyconfigs/h/hwloc/hwloc-1.11.3-GCC-5.2.0.eb [hunk 2]
```diff
@@ -16,8 +16,9 @@ toolchain = {'name': 'GCC', 'version': '5.2.0'}
 source_urls = ['http://www.open-mpi.org/software/hwloc/v%(version_major_minor)s/downloads/']
 sources = [SOURCE_TAR_GZ]
 
-dependencies = [('numactl', '2.0.11')]
+#dependencies = [('numactl', '2.0.11')]
+osdependencies = [('numactl')]
 
-configopts = "--enable-libnuma=$EBROOTNUMACTL"
+configopts = "--enable-libnuma="
```
Comment (boegel) [inline easybuild/easyconfigs/h/hwloc/hwloc-1.11.3-GCC-5.2.0.eb]: @shahzebsiddiqui same here, you should not change this...

File: easybuild/easyconfigs/o/OpenMPI/OpenMPI-2.0.0-GCC-5.2.0.eb [hunk 3]
```diff
@@ -14,13 +14,21 @@ source_urls = ['http://www.open-mpi.org/software/ompi/v%(version_major_minor)s/d
...
 configopts += '--enable-mpirun-prefix-by-default '  # suppress failure modes in relation to mpirun path
 configopts += '--with-hwloc=$EBROOTHWLOC '  # hwloc support
 configopts += '--disable-dlopen '  # statically link component, don't do dynamic loading
+configopts += '--with-x ' # enabling X support
+configopts += '--enable-mpi-fortran ' # enable MPI Fortran bindings
+configopts += '--enable-mpi-cxx ' # enable C++ MPI bindings
+configopts += '--with-cma ' # cross memory attach support
+configopts += '--enable-memory-linux-malloc-alignment ' # memory malloc alignment
+configopts += ' --enable-memchecker ' # enable memory and buffer check
+configopts += '--with-valgrind ' # enable valgrind support
```
Comment (boegel) [inline easybuild/easyconfigs/o/OpenMPI/OpenMPI-2.0.0-GCC-5.2.0.eb]: @shahzebsiddiqui more site-specific changes...

**Comment by shahzebsiddiqui** [comment]
I don't understand the error, why can't it resolve the dependency for MPICH/3.2-GCC-5.2.0

I have committed all the dependency in this pr.

**Comment by boegel** [comment]
@shahzebsiddiqui The error is pretty clear, no? The easyconfig file that provides the `MPICH/3.2-GCC-5.2.0` module is not available yet. The `MPICH-3.2-GCC-5.2.0.eb` easyconfig file is not available yet in the `develop` branch, and it is not included in this PR.

The only way to fix that is to include this easyconfig here (for example using `--update-pr`).

That still leaves the problem of the site-specific changes in the existing easyconfigs though...

**Pull request closed without merging**